### PR TITLE
[Testing Audit] Added tests for Drawer subclasses.

### PR DESCRIPTION
**Pull request by jtlan**
Added tests for `Drawer` subclasses. Most of them only define an element name and class name, which is tested [in the superclass tests](https://github.com/palantir/plottable/pull/2976).

screen shot 2015-11-18 at 1 13 50 pm

Part of #2622.

### CODE REVIEW

File: test/components/labelTests.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 describe("Labels", () => {
   const LABEL_CLASS = "label";
-  let CLOSETO_REQUIRMENT;
+  let CLOSETO_REQUIRMENT: number;
```
Comment (ztsai) [inline test/components/labelTests.ts]: weird, isn't this in develop already?

File: test/drawers/arcDrawerTests.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      ];
+      drawer.draw(data, drawSteps);
+
+      assert.strictEqual(drawer.selection().style("stroke"), "none");
+
+      svg.remove();
+    });
+  });
+
+  describe("Arc Outline Drawer", () => {
```
Comment (ztsai) [inline test/drawers/arcDrawerTests.ts]: is there any reason for grouping these 2 drawers in one test while others in their own file?

File: test/components/labelTests.ts (hunk 1 above)
Comment (jtlan) [inline test/components/labelTests.ts]: Not sure. I cherry-picked the commit here before the other PR merged, so it's possible Github got confused.

File: test/drawers/arcDrawerTests.ts (hunk 2 above)
Comment (jtlan) [inline test/drawers/arcDrawerTests.ts]: Not wanting to have extra files. But I'll split it for consistency.

**Pull request merged**